src/Dialog/Dialog.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import classNames from 'classnames';
+
+class Dialog extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        isOpen: PropTypes.bool
```

Comment:
Yeah, I feel that would be better. It is inconsistent with the other component's style. I initially used isOpen because I thought I should make it different with the native open.

Give me a moment, I'll update the PR.